### DISCUSSION TITLE
Not able to Copy a question set

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#465] [Bugs] [answered]
I'm a user who is having a **Content creator and Book Creator** Role.
I've created a Question set called **Baseline Odia** and published this content.

Now when I open this  **Baseline Odia** content through my profile, I can see there is an option to copy the content.

When I'm trying to make a copy of that **Question set**, getting 400 bad server error, with error message:
"Invalid Props are : [organisation, origin, originData, creator]"
![image](https://user-images.githubusercontent.com/107101132/232680576-13853ffd-204c-4b4a-a864-f81535332734.png)

The version using SunbirdEd is **4.6**.

I'm trying to copy the Question Set Content which is created by me but unable to do that.

I tried to check for the Course content also, but there I'm successfully able to copy the content by following the above steps.

Can anyone help?

cc: @rajeevsathish @sharathkashyap @vinukumar-vs @rhwarrier @reshmi-nair @Krishnaj20

**Comment by rajeevsathish** [comment]
@nidhikachhawa24 which version of the SunbirdED you are using, which Content type you are trying to copy and what is the userrole 

cc: @Krishnaj20 @vinukumar-vs

**Reply by nidhikachhawa24** [reply]
@rajeevsathish 
The version of SunbirdEd is **4.6**.

Trying to copy the **Question set** Content

Also, the user role is the **Content creator and Book Creator**

**Reply by Krishnaj20** [reply]
@nidhikachhawa24  - Please provide more details about the issue as per your debug analysis. Hope you understand that with this information it is extremely difficult for the community to provide inputs about the issue. Please take help of @vinod303 for getting these details.

@madhucr

**Reply by nidhikachhawa24** [reply]
@Krishnaj20 

I've tried to add more details into this. Hope this will be helpful

**Comment by rajeevsathish** [comment, marked as answer]
@nidhikachhawa24 as this is not implemented in sunbirdED with the inQuiry in the current release we cannot copy any QuestionSet. this will be taken in the future release. or you can contribute for this and we can help you on the contribution.

cc: @Krishnaj20 @vinukumar-vs
